### PR TITLE
Update GoDaddy.xml

### PR DESCRIPTION
**Pull request by koops76**
#10782

### CODE REVIEW

File: src/chrome/content/rules/GoDaddy.xml [hunk 1]
```diff
@@ -1,22 +1,4 @@
...
-Non-2xx HTTP code: http://au.godaddy.com/ (200) => https://au.godaddy.com/ (403)
-Non-2xx HTTP code: http://dk.godaddy.com/ (200) => https://dk.godaddy.com/ (403)
-Non-2xx HTTP code: http://fr.godaddy.com/ (200) => https://fr.godaddy.com/ (403)
-Non-2xx HTTP code: http://it.godaddy.com/ (200) => https://it.godaddy.com/ (403)
-Non-2xx HTTP code: http://no.godaddy.com/ (200) => https://no.godaddy.com/ (403)
-Non-2xx HTTP code: http://pk.godaddy.com/ (200) => https://pk.godaddy.com/ (403)
-Non-2xx HTTP code: http://pt.godaddy.com/ (200) => https://pt.godaddy.com/ (403)
-Non-2xx HTTP code: http://sg.godaddy.com/ (200) => https://sg.godaddy.com/ (403)
-Non-2xx HTTP code: http://fi.godaddy.com/ (200) => https://fi.godaddy.com/ (403)
-Non-2xx HTTP code: http://tr.godaddy.com/ (200) => https://tr.godaddy.com/ (403)
```
Comment (J0WI) [inline src/chrome/content/rules/GoDaddy.xml]: I can't confirm all the errors and the ruleset is still off.

Comment (koops76) [inline src/chrome/content/rules/GoDaddy.xml]: @J0WI Please tell which removed targets are working so I will re-add them.

Comment (koops76) [inline src/chrome/content/rules/GoDaddy.xml]: @J0WI The ruleset is re-enabled.

Comment (J0WI) [inline src/chrome/content/rules/GoDaddy.xml]: Basically all of them, see https://github.com/EFForg/https-everywhere/pull/10944#issuecomment-313884041

Comment (koops76) [inline src/chrome/content/rules/GoDaddy.xml]: @J0WI Done.

**Comment by Hainish** [comment]
This is Akamai as well, just as in #10944:

```shell
user@https-everywhere ~ $ dig +short http://pt.godaddy.com/
user@https-everywhere ~ $ dig +short pt.godaddy.com                                                                                                   
wildcard.godaddy.com.edgekey.net.
e6001.x.akamaiedge.net.
23.50.34.176
user@https-everywhere ~ $ dig +short -x 23.50.34.176
a23-50-34-176.deploy.static.akamaitechnologies.com.
```

**Comment by koops76** [comment]
@J0WI I added the ruleset into the whitelist.

**Comment by Hainish** [comment]
Let's wait until we figure this out in #10944.  I'd like to avoid adding things to the whitelist unless absolutely necessary, and we may be able to solve this another way.

File: utils/ruleset-whitelist.csv [hunk 2]
```diff
@@ -1710,6 +1710,7 @@ cb7a292994a2fc87f1440d181779f6301b3483c21e5df5c3d7e9ffaaff78770c,1,1,GNOME.xml
 5ddfa3a482674b780d4e0cd81e3ba0264703fe2b42c797a132127a132df0f974,1,1,Gnumeric.org.xml
 25d4ef946d891943dc472eedd8411e05c4ba679047a1b058f9aac1c2611d0cd9,1,1,GoAruna.com.xml
 d71b4a621557b40535fbc3c3d25ac92b0529a901b82e9ae46bcd939451ba9f97,1,1,GO.com.xml
+660db2609d3abb3b65508b3a1be7cbb60fc2a1d82309709e5da013a0b009d278,0,1,GoDaddy.xml
```
Comment (Hainish) [inline utils/ruleset-whitelist.csv]: Please remove this so we can re-test with the fix applied in https://github.com/EFForg/https-everywhere/commit/db8d6495dddb1c954faf2c9f4b65d4320a873e86

Comment (koops76) [inline utils/ruleset-whitelist.csv]: @Hainish Done.

**Comment by Hainish** [comment]
Looks good to me, I'll hand this back to @J0WI for further review and merging.

**Pull request merged**